integration.go

```diff
@@ -19,6 +21,13 @@ type AWSIntegrationFragment struct {
 	OwnershipTagKeys     []string `graphql:"ownershipTagKeys"`
 }
 
+type AzureResourcesIntegrationFragment struct {
+	TenantId       string        `graphql:"tenantId"`
+	SubscriptionId string        `graphql:"subscriptionId"`
+	LastSyncedAt   *iso8601.Time `graphql:"lastSyncedAt"`
```

Comment:
i don't see this field in the public API

Reply:
Same, this is out